.readthedocs.yml

```diff
@@ -1,7 +1,24 @@
-version: 1
+# .readthedocs.yml
+# readthedocs configuration file
+# see https://docs.readthedocs.io/en/stable/config-file/v2.html for details
 
+# required
+version: 2
+
+# conda setup
 conda:
-    file: .environment.yml
+  environment: .rtd_conda_env.yml
 
+# python requirements
 python:
-    setup_py_install: true
+  version: 3.7
```

Comment:
explanation: this was before only defined in the web interface of readthedocs, which leads to hard-to-track behavior, thus it is preferable to define everything here in the .yml file.